third_party/nvfuser/csrc/scheduler/utils.cpp

```diff
@@ -2116,6 +2141,9 @@ DisjointSets<IterDomain*> disjointViewSets(Fusion* fusion) {
         auto split = expr->as<Split>();
         disjoint_view_ids.mapEntries(split->in(), split->inner());
```

Comment:
Consider renaming this variable to `disjoint_rfactor_ids`, and this function to `disjointRFactorSets `

Reply:
Done: c4d6b39c0a5559696e6c385196a44eca9b7a4b2a